File: Arrays/2114. Maximum Number of Words Found in Sentences.py
def mostWordsFound(sentences):
    max = 0
    for i in range(len(sentences)):
        temp = sentences[i].split(' ') # temo is an array
        if len(temp)>max:
            max = len(temp)
    return max

print(mostWordsFound(["alice and bob love leetcode", "i think so too", "this is great thanks very much"]))

File: Arrays/2114. Maximum Number of Words Found in Sentences.js
link = 'https://leetcode.com/problems/maximum-number-of-words-found-in-sentences/'


// Solution
//---------------------------------
const mostWordsFound = (sentences) => {
    let max = 0
    for (let i = 0; i < sentences.length; i++) {
        let temp = sentences[i].split(' ')
        if(temp.length > max){
            max = temp.length
        }
    }
    return max
}
//---------------------------------
let result = mostWordsFound(["please wait", "continue to fight", "continue to win"])
console.log(result)